BrowserKit/Sources/Shared/UserAgent.swift

```diff
@@ -169,10 +169,10 @@ public struct UserAgentBuilder {
     public static func defaultMobileUserAgent() -> UserAgentBuilder {
         return UserAgentBuilder(
             product: UserAgent.product,
-            systemInfo: "(\(UIDeviceDetails.model); CPU iPhone OS \(UIDeviceDetails.systemVersion.replacingOccurrences(of: ".", with: "_")) like Mac OS X)",
+            systemInfo: "(\(UIDeviceDetails.model); CPU iPhone OS 18_7 like Mac OS X)",
```

Comment:
Updated from 18_6 to 18_7 as WebKit published some internal changes now that reveal they were actually tracking the newer versions for the freeze, presumably for some of the next dot 26.x releases, so the "constant" seems not constant enough for now, more of a moving target.

Reply:
(Confirmed as of 26.0.1 now the constant is at `18_7` in Safari.)

Reply:
With 18.7.2 out, 18.7.2 identifies itself still as 18_7_2 …

However, 26.1 keeps using 18_7, as per: [`WebKit/WebCore` /page/Quirks.cpp#L1859-L1867](https://github.com/WebKit/WebKit/blob/6f96a5e/Source/WebCore/page/Quirks.cpp#L1859-L1867)